tests/requirements.txt

```diff
@@ -4,29 +4,46 @@
 #
```

Comment:
Add `tests/requirements.in` and include main `requirements.in` in it.

Reference: https://github.com/fabric8-analytics/fabric8-analytics-utils/blob/5a5ce60c38c042a43733e7433aa01c71d16542f1/tests/requirements.in#L5